.github/ISSUE_TEMPLATE/bug_ops.md

```diff
@@ -0,0 +1,63 @@
+---
+name: 🐞 Bug
+about: File a bug/issue
+title: '[BUG] <title>'
+labels: bug, ops
+assignees: ''
+
+---
+
+<!--
+Note: Please search to see if an issue already exists for the bug you encountered.
+-->
+
+ 
+**Environment:**
+<!-- 
+- Platform: IVV Orange Cloud
+- Configuration:
+-->
+  
+**Test:**
+<!-- 
+- Name: TST_INFRA_DEP_orange
+- Traçability (requirements): NA
+-->
+  
```

Comment:
Is this test part still needed for OPS ?

Reply:
No need, deleted